build-logic/src/main/kotlin/kotlinx/io/conventions/multiplatform-lib-conventions.gradle.kts

```diff
@@ -95,3 +99,65 @@ fun NamedDomainObjectContainer<KotlinSourceSet>.createSourceSet(
     return result
 }
 
+val versionCatalog: VersionCatalog = extensions.getByType<VersionCatalogsExtension>().named("libs")
+
+kotlin {
+    jvmToolchain {
+        val javaVersion = versionCatalog.findVersion("java").getOrNull()?.requiredVersion
+            ?: throw GradleException("Version 'java' is not specified in the version catalog")
+        languageVersion.set(JavaLanguageVersion.of(javaVersion))
+    }
+
+    jvm {
+        withJava()
+        testRuns["test"].executionTask.configure {
+            useJUnitPlatform()
+        }
+    }
+
+    js(IR) {
+        browser {
+            testTask {
+                filter.setExcludePatterns("*SmokeFileTest*")
+            }
+        }
+    }
+
+
+    sourceSets {
+        named("commonTest") {
```

Comment:
minor: AFAIK `commonTest {}` should be resolvable out of the box, isn't it?

Reply:
Yes, thanks